python/main.py

```diff
@@ -48,23 +91,27 @@ async def add_item(name: str = Form(...), category: str = Form(...), image: Uplo
     logger.info(f"Receive item, name: {name}, category: {category}")
     
     hashedImageName = await hashImage(image)
-    newItem = {
-        'name': name,
-        'category': category,
-        'image_filename': hashedImageName
-    }
-    allItems = getItems()
-    allItems["items"].append(newItem)
-
-    with open(fileName, 'w') as itemsFile:
-        json.dump(allItems, itemsFile)
+    postDbItem(name, category, hashedImageName)
 
     return {"message": f"item received with name: {name}, category: {category}, image: {hashedImageName}"}
 
 @app.get("/items")
 def get_items_reponse():
-    allItems = getItems()
-    return allItems
+    allItems = getDbItems()
+    if len(allItems) > 0:
+        returnItems = formatItemsForReturn(allItems)
+        return returnItems
+    else:
+        return "database is empty"
+
+@app.get("/search")
+def get_searched_response(keyword: str):
+    matches = searchForDbItem(keyword)
+    if len(matches) > 0:
+        returnItems = formatItemsForReturn(matches)
+        return returnItems
+    else:
+        return f"No matches found for: {keyword}"
```

Comment:
Consider returning status code 404 if you want to see no matching records as an error!
Ref: 
https://fastapi.tiangolo.com/advanced/additional-status-codes/